src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.ValueGenerator/Main.cs

```diff
@@ -110,7 +130,33 @@ public List<Result> Query(Query query)
             }
             catch (FormatException e)
             {
-               Log.Debug(GetTranslatedPluginTitle() + ": " + e.Message, GetType());
+                Log.Debug(GetTranslatedPluginTitle() + ": " + e.Message, GetType());
+                if (!string.IsNullOrEmpty(query.ActionKeyword))
+                {
+                    return GetSuggestionFuzzyResults(query, results);
+                }
+            }
+
+            return results;
+        }
+
+        private List<Result> GetSuggestionResults(Query query, List<Result> results)
+        {
+            foreach (var generatorData in QueryHelper.GeneratorDataList)
+            {
+                results.Add(new Result
+                {
+                    Title = generatorData.GetResultTitle(),
+                    SubTitle = generatorData.GetResultSubtitle(),
+                    IcoPath = GetIcoPath(),
+                    ToolTipData = new ToolTipData(generatorData.GetResultTitle(), generatorData.GetResultSubtitle()),
+                    ToolTipVisibility = Visibility.Visible,
+                    Action = c =>
+                    {
+                        _context.API.ChangeQuery($"{query.ActionKeyword} {generatorData.Keyword}");
+                        return false;
+                    },
```

Comment:
To fix the wrong query override replace this code with `QueryTextDisplay`. The `Action` is invoked on pressing enter and `QueryTextDisplay` updates the query on selecting the result.

Reply:
If you will keep the press enter behavior and I personally like it, it might make sense to set the second parameter of `ChangeQuery()` to true.